server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/pipeline/stages/FinalResult.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.engine.pipeline.stages;
+
+import lombok.Value;
+import org.eclipse.lsp.cobol.common.model.tree.Node;
+import org.eclipse.lsp.cobol.common.symbols.SymbolTable;
+
+import java.util.Map;
+
+/**
+ * Final processing result
+ */
+@Value
+public class FinalResult {
```

Comment:
The name is too general. ProcessingResult? CodeAnalysisResult?

Reply:
Fixed